CONTRIBUTING.md

```diff
@@ -30,6 +30,7 @@ The extension recevies logs in the form of `object` from activityLog API. The ac
 
 - [_`src`_](https://github.com/mozilla/extension-activity-monitor/tree/master/src) - Inside _src_ the extension's code can be found (manifest.json, background.js etc.)
   - [_`activitylog`_](https://github.com/mozilla/extension-activity-monitor/tree/master/src/activitylog) - Contains activity log page (extension page).
+  - [_`devtools`_](https://github.com/mozilla/extension-activity-monitor/tree/master/src/devtools) - Contains devtools panel.
```

Comment:
@atiqueahmedziad  the "src/devtools" does actually contain the invisible devtools page that **will create** a devtools panel which loads the activitylog.html page with a tabId filter set to `browser.devtools.inspectedWindow.tabId`.

Reply:
Is it okay if I write - "Contains the invisible devtools page that creates a devtools panel which loads the activitylog.html page and filter logs with the current tab id" ?